src/internal/serviceenv/config.go

```diff
@@ -25,13 +25,17 @@ type GlobalConfiguration struct {
 	LokiHost                       string `env:"LOKI_SERVICE_HOST"`
 	LokiPort                       string `env:"LOKI_SERVICE_PORT"`
 	OidcPort                       uint16 `env:"OIDC_PORT,default=1657"`
+	IsPachw                        bool   `env:"IS_PACHW,default=false"`
+	PachwEnabled                   bool   `env:"PACHW_ENABLED,default=true"`
+	PachwMinReplicas               int    `env:"PACHW_MIN_REPLICAS,default=0"`
```

Comment:
I don't think setting the default to zero is needed since that is the default value for an int.